.changeset/calm-laws-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/dropdown-menu': minor
+---
+
+Allowed `data-*` attributes to be routed down the component
```

Comment:
Now, with the accessible-button change you'll have to add to add it to the changeset as well.

Regarding the wording, I'd suggest to use "Supports forwarding" (... data-attributes to [component(s)] ... ) instead of "allows routing".

Reply:
I have updated the changeset file in this commit: https://github.com/commercetools/ui-kit/pull/2855/commits/22313d616d558c6ca97fc06bb8f9b2875a330f9c